modules/installation-requirements-user-infra.adoc

```diff
@@ -137,6 +137,9 @@ ifndef::ibm-z,ibm-power[|{op-system} or RHEL 7.6]
 |8 GB
 |120 GB
 
+5+a|
+^1^ 2 vCPUs are equivalent to 1 core when hyper-threading is enabled. 1 vCPU is equivalent to 1 core when hyper-threading is not enabled.
```

Comment:
I am not an expert on CPU hardware stuff. But I have some concern, is there some existing technical theory support this statement? The statement sound like disabling hyper-threading will has more powerful CPU resource, but per QE's testing, once disable hyper-threading, will need more vCPUs to meet a cluster install requirement.

Reply:
@jianlinliu thank you for your review.

From a technical standpoint, my understanding is that if Intel hyper-threading is enabled, which I think it is by default on Intel based systems for OpenShift, each physical CPU core is divided into two logical cores (vCPUs) that can each handle a processing thread. So, if you have it enabled, 1 physical core provides logical cores that can each handle a processing thread. If it is not, then 1 physical core provides only one logical core that can handle one processing thread at a time.

Regarding performance, I can see your point about how the sentence could be interpreted to mean that you only get one core's worth of CPU per 2 vCPUs if hyper-threading is enabled. I think in reality it is the other way round, you get two logical processing units for each core and if your program can divide CPU requests into multiple tasks that can be handled in parallel, then this reduces bottlenecks in the processing pipeline. The core is then more efficient because it is idle less often.

Does this alternative sentence resolve the issue?

"1 physical core provides 2 vCPUs when hyper-threading is enabled. 1 physical core provides 1 vCPU when hyper-threading is not enabled."

Thanks.

Reply:
That sounds better to me.

Reply:
Great! I have subsequently updated the pull request to include that sentence.

Reply:
Dropping in to say that this is a great rephrasing!

Reply:
Thanks Bob!